### DISCUSSION TITLE
James D. McCaffrey PyTorch blogs

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [dotnet/TorchSharpExamples#37] [Ideas]
[James D. McCaffrey's](https://github.com/jdmccaffrey) [PyTorch Blogs](https://jamesmccaffrey.wordpress.com/category/pytorch/) are featured in the  [.NET insight](https://visualstudiomagazine.com/Home.aspx)

![image](https://user-images.githubusercontent.com/49812372/191837504-d71cd5b2-d8e5-4a6d-90f4-1cb7f4b9389c.png)

These PyTorch blogs could provide inspiration for porting PyTorch examples to TorchSharp

- [ ] [Multi-Class Classification Using PyTorch, Part 1: New Best Practices” in Visual Studio Magazine](https://jamesmccaffrey.wordpress.com/2022/09/15/multi-class-classification-using-pytorch-part-1-new-best-practices-in-visual-studio-magazine/)
- [ ] [Multi-Class Classification Using New PyTorch Best Practices, Part 2: Training, Accuracy, Predictions](https://visualstudiomagazine.com/articles/2022/09/12/multi-class-pytorch-2.aspx)

**Comment by GeorgeS2019** [comment]
### [A Quick Look at PyTorch Audio](https://jamesmccaffrey.wordpress.com/2022/08/30/a-quick-look-at-pytorch-audio/)

> The demo code is relatively self-explanatory but has many subtleties. The demo downloads the YesNo audio dataset. The dataset consists of 60 recordings of a man saying “yes” or “no” in Hebrew eight times. The labels are 0 for “no” and 1 for “yes”. The pattern of yes and no is random. For example, recording [3] in the demo is “no, no, yes, no, no, no, yes, no”.

Torchsharp provides [YesnoDataset](https://github.com/dotnet/TorchSharp/blob/main/src/TorchSharp/TorchAudio/Datasets/YesnoDataset.cs)

> In my opinion, machine learning for audio data has evolved into an area that is distinct and quite different from ML for image data, ML for natural language, and ML for tabular data.


#kaiidams  FYI

**Comment by GeorgeS2019** [comment]
### [Yes, TensorFlow is Dead ??](https://jamesmccaffrey.wordpress.com/2022/07/29/yes-tensorflow-is-dead/)

What do others think about that?
